waku/v2/node/rest/store/openapi.yaml

```diff
@@ -0,0 +1,91 @@
+openapi: 3.0.3
+info:
+  title: Waku V2 node REST API
+  version: 1.0.0
+  contact:
+    name: VAC Team
+    url: https://forum.vac.dev/
+
+tags:
+  - name: store
+    description: Store REST API for WakuV2 node
+
+paths:
+  /store/v1/messages:
+    get:
+      summary: Gets message history
+      description: >
+        Retrieves WakuV2 message history. The returned history
+        can be potentially filtered by optional request parameters.
+      operationId: getMessageHistory
+      tags:
+        - store
+      responses:
+        '200':
+          description: WakuV2 message history.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/StoreResponse'
+        '4XX':
+          description: Bad request error.
+        '5XX':
+          description: Unexpected server error.
+
+components:
+  schemas:
+    StoreResponse:
+      type: object
+      properties:
+        messages:
+          type: array
+          items:
+            $ref: '#/components/schemas/WakuMessage'
+        paging_options:
+          $ref: '#/components/schemas/PagingOptions'
+      required:
+        - messages
+
+    PagingOptions:
+      type: object
+      properties:
+        page_size:
+          type: integer
+          format: int32
+        cursor:
+          $ref: '#/components/schemas/Index'
+        forward:
+          type: boolean
+        required:
+          - page_size
+          - forward
+
+    Index:
+      type: object
+      properties:
+        digest:
+          type: string
+        received_time:
+          type: integer
+          format: int64
+        required:
+          - digest
+          - received_time
+
+    WakuMessage:
+      type: object
+      properties:
+        payload:
+          type: string
+        content_topic:
+          type: string
+        version:
+          type: integer
+          format: int32
+        timestamp:
+          type: integer
+          format: int64
+        ephemeral:
+          type: boolean
+      required:
+          - payload
```

Comment:
```suggestion
      required:
          - content_topic
          - payload
```

Reply:
Corrected locally. In the next commit I will also add the missing info, i.e. query-params with description/examples, that we commented in our 1:1 call.